views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,7 @@
+ <img src={{image_url}} alt="imgPic">
```

Comment:
Since some beers do not have images and you can see the image icon with the alt text, you could consider wrapping this line in an if statement, so that we can only see the image if there is one.